OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -830,6 +848,23 @@ static void ShowStatusBarsDropdown(DropDownButtonWidget dropdown, GameSettings s
 			dropdown.ShowDropDown("LABEL_DROPDOWN_TEMPLATE", 500, options.Keys, setupItem);
 		}
 
+		static void ShowDisplaySelectionDropdown(DropDownButtonWidget dropdown, GraphicSettings s)
+		{
+			var options = Exts.MakeArray(Game.Renderer.DisplayCount, i => string.Format("{0} {1}", DisplayBaseName, i));
+
+			Func<string, ScrollItemWidget, ScrollItemWidget> setupItem = (o, itemTemplate) =>
+			{
+				var item = ScrollItemWidget.Setup(itemTemplate,
+					() => s.VideoDisplay == options.IndexOf(o),
```

Comment:
Using `IndexOf` to work backwards to find the index isn't great - please use my previous suggestion to pass the offsets themselves to `setupItem` instead.

IMO the label text should also be using the index + 1 instead of counting from 0.

Reply:
Changed it to count from 1.

Reply:
I admit I did not understand what you were getting at. Now I do and it is done.